src/codegen/ast_interpreter.cpp

```diff
@@ -75,6 +75,12 @@ class RegisterHelper {
     static void deregister(void* frame_addr);
 };
 
+/*
+ * ASTInterpreters exist per function frame - there's no global interpreter object that executes
+ * all non-jitted code!
+ *
+ * The ASTInterpreter inherits from Box as part of generator support.
```

Comment:
this likely won't hold true upon rebasing (or at least in the future) - with the GCWrapper stuff @undingen is working on.  But I'd s/generator support/GC support/ or something to that effect.